File: api/Models/SlotMachine/SlotSymbol.cs
namespace WebApiModels.SlotMachine
{
    public class SlotSymbol
    {
        public SymbolType Symbol { get; set; }
        public int Currency { get; set; }
    }

    public enum SymbolType
    {
        Castle = 1,
        King = 2,
        Queen = 3,
        Knight = 4,
        Crown = 5,
        Swords = 6,
        Gerb = 7
    }
}

File: client/src/components/SlotMachine/SlotGame/SlotGameContent/SlotColumn/SlotSymbol/SlotSymbol.js
import React from 'react'

import { castle, king, queen, knight, gerb, crown, swords } from '../../../../images';

import './SlotSymbol.css';

const SlotSymbol = props => {
  const { symbol } = props;

  const takeImage = () => {
    switch (symbol) {
      case 1:
        return castle;
      case 2:
        return king;
      case 3:
        return queen;
      case 4:
        return knight;
      case 5:
        return crown;
      case 6:
        return swords;
      case 7:
        return gerb;
      default:
        window.alert('Cant get symbol image!');        
    }
  }
  
  return (
    <div className="SlotSymbolWrapper">
      <img
        alt="slot symbol"
        src={takeImage(symbol)}
      />
    </div>
  )
}

export default SlotSymbol;